terminology/README.md

```diff
@@ -0,0 +1,15 @@
+# Terminology
+
+The language we use has power. Being as concise and descriptive as possible helps to lessen ambiguity. This leads to shared understanding and better outcomes.
+
+## Terms
+
+* Use `blocklist`, and not `blacklist`.
+* Use `allowlist`, and not `whitelist`.
+* Avoid using `master` and `slave`. 
+	* For git, use `main`.
+	* For databases, use `write database` and `read replica`.
+	For other scenarios, use terms such as `parent` and `child` or `leader` and `follower`. Be mindful of scenarios where other commands will create language like, sending `kill` signals to `children`.
```

Comment:
Should this line have another `*` before it?

Reply:
Ah, good catch, thank you. Updated.